corehq/tests/tools.py

```diff
@@ -0,0 +1,4 @@
+
+def nottest(fn):
```

Comment:
Why not `not_test`? Or `not_a_test`?

Reply:
The name originally came from `nose.tools`, which we did not write.